src/autoqasm/converters/arithmetic.py

```diff
@@ -0,0 +1,68 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+"""Converters for aritmetic operator nodes"""
+import ast
+
+import gast
+from malt.core import ag_ctx, converter
+from malt.pyct import templates
+
+ARITHMETIC_OPERATORS = {
+    gast.Fd : "ag__.fd",
```

Comment:
I see your unit test is failing with:
```AttributeError: module 'gast' has no attribute 'Fd'```
I believe the `gast` node for this is called `gast.FloorDiv`. Are you able to run the tests locally via `tox -e unit-tests`? Or, for faster iteration on your changes, you can install your local version of `autoqasm` by running `pip install -e .[test]` from the repo root, and then `pytest` will run your code and tests. I find this to be the fastest way to iterate locally.

Reply:
1. ok. to be honest unit testing is something i have not done much so kind of overwhelming for me. I started with
tox -e unit-tests -- test_operators::test_arithmetic_fd
this command. So I am afraid I'll be bothering you with very basic questions. and I am really sorry for that. 
2. Have you checked two new files I added? Are those correct?

Reply:
No worries, thank you for the contribution.

What was the result when you ran the tox command? Do you need help understanding the output?

Tox should tell you whether your tests are passing. If the test itself is good, that helps give us confidence that the two new files are correct. I left one comment on the test. 👍

Reply:
The PR page also shows the results of running both the unit tests and the style linters. If it feels overwhelming to run the tests, you can start by inspecting the details pages on the PR (such as [here](https://github.com/amazon-braket/autoqasm/actions/runs/9416716530/job/25947089446?pr=26) and [here](https://github.com/amazon-braket/autoqasm/actions/runs/9416716625/job/25947088003?pr=26)).

As you become familiar with the output, it will be more efficient to move to running the tests locally, like @rmshaffer said.

Reply:
@daredevil3435 I get that the test framework can be a bit complicated! It may be less overwhelming to start by just testing your changes manually, for example, using a local Python program or a Jupyter notebook. Try to construct an AutoQASM program using the `//` operator, build it, and see if your code runs - and if not, what error message you would get. For example, you could put this in a Jupyter notebook cell:
```python
    import autoqasm as aq

    @aq.main
    def prog():
        a = aq.IntVar(5)
        b = aq.IntVar(2)
        c = a // b

    prog.build().display()
```

If successful, this should print the OpenQASM program. If it fails, it should give you an error message that points back to your code somewhere. (If you're hitting an error message and you don't know what it means, feel free to paste the error here and ask for help!)